File: 236A.cpp
#include<iostream>
#include<unordered_set>
#include<vector>
#include<cmath>
#include<algorithm>
#include<string>

using namespace std;


int main()
{
   string a;
   unordered_set <char> b; 
   cin >> a;
   for(char c : a)
   {
       b.insert(c);
   }
   if(b.size()%2 ==0)
   {
       cout << "CHAT WITH HER!";
   }
   else
   {
       cout << "IGNORE HIM!";
   }
   
}

File: 236A.rb
require 'set'
s = gets.chomp

s1 = Set.new

for i in 0...s.length
    s1.add(s[i])
end

if s1.length % 2 == 0
    puts "CHAT WITH HER!"
else
    puts "IGNORE HIM!"
end

